multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -87,7 +87,7 @@ def parse_genome_results(self, f):
         'median_insert_size': r"median insert size = ([\d,\.]+)",
         'mean_mapping_quality': r"mean mapping quality = ([\d,\.]+)",
         'general_error_rate': r"general error rate = ([\d,\.]+)",
-        'mean_coverage': r"mean coverageData = ([\d,]+)"
+        'mean_coverage': r"mean coverageData = ([\d]+\.[\d]+)"
```

Comment:
You've dropped the comma here, which will break it for average coverages > 1000X. I think it's better to just use the same regex as above, with the additional `.` allowed?

```suggestion
        'mean_coverage': r"mean coverageData = ([\d,\.]+)"
```